docs/getting-started/java.md

```diff
@@ -0,0 +1,226 @@
+</dependency>
+```
+
+</TabItem>
+<TabItem value="groovy" label="Gradle Groovy DSL">
+
+```groovy
+implementation 'dev.openfeature:sdk:1.0.0'
+implementation 'dev.openfeature.contrib.providers:flagd:0.5.1'
+```
+</TabItem>
+</Tabs>
+
+### Step 3: Add code
+
+Our simple web server contains a single endpoint `/hello`. Let's add required codes to get this up and running with
+recommended best practices. 
+
+Please note that you may have your own package names and package naming in sample codes will require alterations.
+
+`OpenFeatureAPI` is the main access point to OpenFeature SDK. This object is designed to act as a singleton. Hence, let's 
+define a simple bean configuration class `OpenFeatureBeans`. This definition allows us to dependency inject `OpenFeatureAPI` 
+singleton to desired Spring components.
+
+```java
+package demo;
+import dev.openfeature.contrib.providers.flagd.FlagdProvider;
+import dev.openfeature.sdk.OpenFeatureAPI;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenFeatureBeans {
+
+    @Bean
+    public OpenFeatureAPI OpenFeatureAPI() {
+        final OpenFeatureAPI openFeatureAPI = OpenFeatureAPI.getInstance();
+        
+        // Use flagd as the OpenFeature provider and use default configurations
+        openFeatureAPI.setProvider(new FlagdProvider());
+
+        return openFeatureAPI;
+    }
+}
+```
+
+Next, we can add our REST endpoint definition with dependency injection for `OpenFeatureAPI` instance,
+
+```java
+package demo;
+import dev.openfeature.sdk.Client;
+import dev.openfeature.sdk.OpenFeatureAPI;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RestHello {
+    private final OpenFeatureAPI openFeatureAPI;
+
+    @Autowired
+    public RestHello(OpenFeatureAPI OFApi) {
+        this.openFeatureAPI = OFApi;
+    }
+    
+    @GetMapping("/hello")
+    public String getHello() {
+        final Client client = openFeatureAPI.getClient();
+
+        // Evaluate welcome-message feature flag
+        if (client.getBooleanValue("welcome-message", false)) {
+            return "Hello, welcome to website.!";
+        }
+
+        return "Hello.!";
+    }
+}
+```
+
+At this point, we are ready to run the initial version of our application.
+
+### Step 4: Run the initial application
+
+Let's compile and run the application.
+
+```shell
+./gradlew clean build
+java -jar build/libs/javaDemo-0.0.1-SNAPSHOT.jar
+```
+
+> Note that you may have a different `.jar` name based on your build configurations.
+
+Now you can visit the url [http://localhost:8080/hello](http://localhost:8080/hello) and observe the message **Hello.!**.
+
+At this point, we are falling back to default response as flagd SDK cannot connect to any feature flag provider. Next, let's 
```

Comment:
Change wording giving the meaning - flagd was selected as the provider and it is already there in the code. But we did not configure the provider, hence by default we fall back to default value

Reply:
Updated in the latest commit